### ISSUE TITLE
Documentation for db._createDatabase is wrong

### ISSUE THREAD

**Issue by karjaneth** [arangodb/arangodb#2977]
## my environment running ArangoDB
I'm using the latest ArangoDB of the respective release series:
- [x] 3.2

Mode:
- [x] Single-Server

Storage-Engine:
- [x] mmfiles

On this operating system:
- [x] Linux 
  - [x] docker - official docker library

**I ran arangodb using the following docker command:**
`$ sudo docker run -e ARANGO_RANDOM_ROOT_PASSWORD=1 --name arangodb-instance arangodb/arangodb:3.2.0`

**Got into arangosh using:**
`$ sudo docker exec -it arangodb-instance arangosh --server.username root --server.password <the generated root password>`

**Tried to create a database using the command [from the docs](https://github.com/arangodb/arangodb/blame/e8f3d72ae55adb71d20839a20059abb0dfdf61a7/Documentation/Books/Manual/DataModeling/Databases/WorkingWith.md#L165):**
`127.0.0.1:8529@_system> db._createDatabase("newDB", [], [{ username: "newUser", passwd: "123456", active: true}])`

**The following error occurs:**

```
JavaScript exception in file '/usr/share/arangodb3/js/client/modules/@arangodb/arango-database.js' at 975,5: ArangoError 400: bad parameter
!    throw new ArangoError(requestResult);
!    ^
stacktrace: ArangoError: bad parameter
    at ArangoDatabase._createDatabase (/usr/share/arangodb3/js/client/modules/@arangodb/arango-database.js:975:11)
    at <shell command>:1:4
```

**Instead I would be expecting:** The database should be created.

**Workaround:**
 I **can** make it work by substituting the `[]` options attribute with `{}`.

```
127.0.0.1:8529@_system> db._createDatabase("newDB", {}, [{ username: "newUser", passwd: "123456", active: true}])
true
```

Looking at [the source file referenced in the error](https://github.com/arangodb/arangodb/blob/e8f3d72ae55adb71d20839a20059abb0dfdf61a7/js/client/modules/%40arangodb/arango-database.js#L968), it does seem that it expects `{}` as the default value for the options, so it would seem that it's the documentation that's wrong.

**Comment by jsteemann** [comment]
Fixed in 3.2.2-

**Comment by jsteemann** [comment]
3.2.1 has been released and is [available for download](https://www.arangodb.com/download-major/).
